nl.js

```diff
@@ -0,0 +1,747 @@
+server.config.jumpMlt Spring Force
+server.config.airStrf Lucht Strafing
+server.config.deltaMlt Tijd Schaal
+server.config.strafeSpd Strafe Snelheid
+server.config.fiRat Vuurkracht
+server.config.reSpd Herlaad Snelheid
+server.config.slidTime Glijd Tijd
+server.config.fixMov Fixed Movement
+server.config.slidSpd Glijd Snelheid
+server.config.healthMlt Gezondheid Vermenigvuldiger
+server.config.impulseMlt Wapen Impuls
+server.config.nameTeam1 Team 1 Naam
+server.config.nameTeam2 Team 2 Naam
+server.config.nameTeam3 Team 3 Naam
+server.config.nameTeam4 Team 4 Naam
+server.config.nameTeam5 Team 5 Naam
+server.config.t1Dmg Team 1 Schade
+server.config.t2Dmg Team 2 Schade
+server.config.t3Dmg Team 3 Schade
+server.config.t4Dmg Team 4 Schade
+server.config.t5Dmg Team 5 Schade
+server.config.selTeam Selecteer Team
+server.config.allowSpect Toeschouwers
+server.config.killRewards Kill Beloningen
+server.config.headshotOnly Alleen Headshots
+server.config.noSecondary Secundaries Uitzetten
+server.config.noStreaks Streaks Uitzetten
+server.config.canSlide Glijden
+server.config.wallJP Muur Spring Kracht
+server.config.autoJump Auto Springen
+server.config.bDrop Kogel Drop
+server.config.thirdPerson 3e Persoon
+server.config.requireCaptcha Captcha Vereist
+server.config.nameTags Verstop Naamplaatjes
+server.config.kCams Kill Camera
+server.config.noReload Niet Herladen
+server.config.private Privé
+server.config.krRewards KR Beloningen
+server.config.forceDefault Dwing Standaard Instellingen
+server.config.warmupTime Opwarm Tijd
+server.config.objtvTime Objectief Tijd
+server.config.forceSpawn Auto Respawn Tijd
+server.config.scoreLimit Score Limiet
+server.config.keepTScore Houd Team Score
+server.config.forceC Dwing Modus Classes
+server.config.logTim Log Tijd (Racen, Parkour)
+server.config.hitBoxPad Hitbox Opvulling
+server.config.hpRegen Gezondheid Regen
+server.config.disableB Borders Uitzetten
+server.config.throwMel Gooibare Melees
+server.config.noCosm Cosmetica Uitzetten
+social.login Login/Registreer
+social.login.failed Login Mislukt
+social.login.buy Login om te Kopen
+social.login.sell Login om te Verkopen
+social.login.gift Login om te Giften
+social.login.proccessing Inloggen...
+social.since Sinds
+social.user.maps Speler Games
+social.user.mods Speler Mods
+social.maps.none Geen Games gevonden...
+social.mods.none Geen Mods gevonden...
+social.market.price.low Laagste Prijs
+social.market.price.list Laagste Prijs in
+social.market.searching Zoeken
+social.market.listed Aangeboden {0} geleden
+social.market.list Bied Item aan
+social.market.unlist Weghalen
+social.market.unlisting Item Weghalen...
+social.market.posting Item Posten...
+social.market.req lvl {0} nodig
+social.market.cant Kan niet Veroorloven
+social.market.purchase Kopen
+social.market.fee Geen List Fee
+social.market.failed Mislukt. Probeer Opnieuw
+social.market.head Hoofd Item
+social.market.back Rug Item
+social.market.waist Waist Item
```

Comment:
'Waist' is 'Taille' in Dutch. It should be changed to 'Taille Item'

Reply:
I changed it.